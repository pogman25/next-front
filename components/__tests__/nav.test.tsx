import React from 'react';
import renderer from 'react-test-renderer';
import Nav from '../nav';
import 'jest-styled-components';

describe('Nav Test', () => {
  test('Nav render', () => {
    const tree = renderer.create(<Nav />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
