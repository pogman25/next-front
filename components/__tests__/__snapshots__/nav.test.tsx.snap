// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Test Nav render 1`] = `
.c0 {
  color: red;
}

<nav
  className="jsx-3111634959"
>
  <ul
    className="jsx-3111634959"
  >
    <li
      className="jsx-3111634959"
    >
      <a
        className="c0"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Home
      </a>
      <a
        className="jsx-3111634959"
        href="/about"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        About
      </a>
      <a
        className="jsx-3111634959"
        href="/swapi"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        SWAPI
      </a>
    </li>
    <li
      className="jsx-3111634959"
    >
      <a
        className="jsx-3111634959"
        href="https://zeit.co/now"
      >
        ZEIT
      </a>
    </li>
    <li
      className="jsx-3111634959"
    >
      <a
        className="jsx-3111634959"
        href="https://github.com/zeit/next.js"
      >
        GitHub
      </a>
    </li>
  </ul>
</nav>
`;
